### PR TITLE
DDF-2323 Quiet the LdapClaimsHandler

### PR DESCRIPTION
**Pull request by harrison-tarr**
#### What does this PR do?
Makes the LDAP log not be as verbose
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@AzGoalie @spearskw 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@kcwire 
@stustison
#### What are the relevant tickets?
DDF-2323
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by AzGoalie** [comment]
👍

**Comment by stustison** [comment]
I'm curious why you'd want this as opposed to just fixing the broken LDAP connection or removing the LDAP claims handler entirely.

**Comment by spearskw** [comment]
👍

File: platform/security/sts/security-sts-attributequerycommon/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java [hunk 1]
```diff
@@ -128,7 +129,11 @@ public ProcessedClaimCollection retrieveClaimValues(ClaimCollection claims,
                 }
             }
         } catch (URISyntaxException e) {
-            LOGGER.error(ERROR_RETRIEVING_ATTRIBUTES, externalAttributeStoreUrl, nameId, e);
+            LOGGER.warn(
+                    ERROR_RETRIEVING_ATTRIBUTES + "Set log level to DEBUG for more information.",
```
Comment (kcwire) [inline platform/security/sts/security-sts-attributequerycommon/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java]: Do we really want to tell an Admin to set the log level to DEBUG? if so I think we should call out the package to set so they don't get overwhelmed with logs.

File: platform/security/sts/security-sts-attributequerycommon/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java [hunk 2]
```diff
@@ -169,7 +176,8 @@ protected ProcessedClaimCollection getAttributes(String nameId) throws URISyntax
                 createClaims(claimCollection, assertion);
             }
         } catch (AttributeQueryException ex) {
-            LOGGER.error("Error occurred in AttributeQueryClient, did not retrieve response.", ex);
+            LOGGER.warn("Error occurred in AttributeQueryClient, did not retrieve response. Set log level to DEBUG for more information.");
```
Comment (kcwire) [inline platform/security/sts/security-sts-attributequerycommon/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java]: Same comment here.

File: platform/security/sts/security-sts-attributequerycommon/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java [hunk 3]
```diff
@@ -264,6 +275,7 @@ protected Service createService() {
                 wsdlURL = uriResolver.isResolved() ? uriResolver.getURL() : new URL(wsdlLocation);
                 service = Service.create(wsdlURL, QName.valueOf(serviceName));
             } catch (Exception e) {
+                LOGGER.warn("Unable to create service from WSDL location. Enable log level to DEBUG for more information.");
```
Comment (kcwire) [inline platform/security/sts/security-sts-attributequerycommon/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java]: also here.

**Comment by kcwire** [comment]
👍

**Comment by stustison** [comment]
👍  as long as you change the other claims handlers to behave similarly or at least write tickets for them

**Pull request merged**